src/services/Store/store/helpers.ts

```diff
@@ -102,3 +106,39 @@ export const canImport = (toImport: Partial<LocalStorage>, store: LocalStorage)
     return diff.length === 0;
   }
 };
+
+export const migrateConfig = (toImport: Partial<LocalStorage>) => {
+  return {
+    ...toImport,
+    // @ts-expect-error rates are present in settings on data to be migrated, want to move it at root of persistence layer
+    rates: toImport.settings?.rates ? toImport.settings.rates : toImport.rates,
+    trackedAssets: toImport.trackedAssets ? toImport.trackedAssets : {},
+    // @ts-expect-error rates are present in settings on data to be migrated, want to move it at root of persistence layer
+    settings: toImport.settings?.rates ? dissoc('rates', toImport.settings) : toImport.settings
+  } as LocalStorage;
+};
```

Comment:
We really want to check the version number and run the existing `migrations`.
If you create a ch card, we can handle in a separate PR and prioritise it next sprint.

Reply:
https://app.clubhouse.io/mycrypto/story/8289/remove-migrateconfig